lib/schemas/api.js

```diff
@@ -1,9 +1,3 @@
 const joi = require('joi')
 
-module.exports = joi.object().keys({
-  controllers: joi.object().required(),
-  models: joi.object().required(),
-  policies: joi.object().required(),
-  services: joi.object().required(),
-  config: joi.object().optional()
-})
+module.exports = joi.object().required()
```

Comment:
For me it should look like : 

```
module.exports = joi.object().keys({
  controllers: joi.object().required(),
  models: joi.object().required(),
  policies: joi.object().required(),
  services: joi.object().required(),
  config: joi.object().optional()
}).unknow()
```
It force to have correct API structure and allow extra keys :)